AggregationDemo.java

```diff
@@ -0,0 +1,44 @@
+package com.basicsstrong.oop;
+
+import java.util.ArrayList;
+import java.util.List;
+
+class College
+{
+	String name;
+	private List<Teacher> teacher;
+	public College(String name,List<Teacher> teacher){
+		this.name=name;
+		this.teacher=teacher;
+	}
+	public List<Teacher> getTeachers()
+	{
+		return teacher;
+	}
+}
+class Teacher{
+	String name;
+	String subject;
```

Comment:
What if a teacher teaches multiple subjects ?

Reply:
Then subjects will be of type array list of strings